### DISCUSSION TITLE
Program freeze at launch!

### DISCUSSION THREAD

**Discussion by alejo9010** [Squirrel/Squirrel.Windows#1818] [General]
hello, im having problem, it seems like its looping to get the update, and wont load my program at all 
this is the line 

            using (var manager = UpdateManager.GitHubUpdateManager("https://github.com/USER/REPOSITORY"))
            {
                await manager.Result.UpdateApp();
            }

but program get stuck, idont know hwere to look at now!

**Comment by alejo9010** [comment]
it get stuck in this line 
`var response = await client.GetAsync(releasesApiBuilder.ToString());`

`UpdateManager.Factory.cs`

my releaseApiBuilder looks like this `{repos/alejo9010/Data/releases}`

**Reply by owns** [reply]
Have you checked the log file in SquirrelTemp (the name is something like that) in the install folder parent?

**Reply by owns** [reply]
https://github.com/Squirrel/Squirrel.Windows/blob/develop/docs/faq.md#installing
